### ISSUE TITLE
Feedback form: Remove redundant aria-required

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/exhibits#2930]
Currently, there are two required input fields in the Feedback form: the **Message** `<textarea>` and the **Email** `<input>`.

These elements have attributes `required="required"` and `aria-required="true"`.

```
<textarea rows="4" required="required" aria-required="true" class="form-control" name="contact_form[message]" id="contact_form_message"></textarea>
```

```
<input required="required" aria-required="true" class="form-control" type="email" name="contact_form[email]" id="contact_form_email">
```

The `aria-required` attribute is redundant and should be removed.

### Reported by SODA

> **Description**
> The element has missing or incorrect states or properties that are necessary for screen reader users to interact with or understand the content conveyed by the element.
> The required elements use both aria-required and html5 required attribute. Use only the html5 required attribute.
> 
> **Recommended to fix**
> Remove the aria-required attribute, 
> 
> `<textarea rows="4" required="required" class="form-control" name="contact_form[message]" id="contact_form_message"></textarea>`
> 
> 
> RULE : 
> The name, role, value, states, and properties of user interface components MUST be programmatically determinable by assistive technologies.
> 
> HOW TO FIX: 
> Fix this issue by ensuring that all necessary ARIA states or properties are used per the ARIA Recommendation document and the ARIA Authoring Practices document (see References below).
> 
> REFERENCE: 
> WAI-ARIA Recommendation: https://www.w3.org/TR/wai-aria-1.1/
> WAI-ARIA Authoring Practices: https://www.w3.org/WAI/ARIA/apg/
> 
> BACKGROUND: 
> Every user interface control must have a role along with any applicable states and properties so that screen reader users know how to interact with the control. Native HTML elements - such as <button>, <a>, <input>, <select> - already have a role, and their necessary states and properties - such as the checked/unchecked state of a checkbox - are automatically conveyed so nothing more needs to be done. If you create a custom version of a native HTML element or a custom control or widget that does not have a native HTML equivalent, you must add the relevant role(s) and any applicable states and properties using ARIA as well as expected keyboard interactions.
